pkg/chaosctl/cmd/logs.go

```diff
@@ -83,7 +82,7 @@ Examples:
 
 // Run logs
 func (o *logsOptions) Run(args []string) error {
-	client, cancel, err := common.CreateClient(context.TODO(), managerNamespace, managerSvc)
+	client, cancel, err := cm.CreateClient(context.TODO(), managerNamespace, managerSvc)
```

Comment:
also duplicated with https://github.com/chaos-mesh/chaos-mesh/pull/3229;

I am very sorry, I know it sounds unfair, but I think that PR brings better work.

The bugbash activity has no assign-like policy, it brings the conflicts.

I am sorry, I hope I could get your forgiveness.